transfuse-core/src/main/java/org/androidtransfuse/gen/FilerSourceCodeWriter.java

```diff
@@ -49,21 +51,23 @@ public OutputStream openBinary(JPackage jPackage, String fileName) throws IOExce
         //generate a source file based on package and fileName
         String qualified = toQualifiedClassName(jPackage, fileName);
         JavaFileObject sourceFile = filer.createSourceFile(qualified, originating.getOriginatingElements(qualified));
-
-        OutputStream os = sourceFile.openOutputStream();
+        OutputStream os = getWriterOutputStream(sourceFile.openWriter());
         openStreams.add(os);
 
         return os;
     }
 
+    public OutputStream getWriterOutputStream(Writer writer) {
+        return new WriterOutputStream(writer, StandardCharsets.UTF_8);
```

Comment:
@johncarl81 using StandardCharsets.UTF_8 this actually red on my editor. it required
                    <source>1.6</source>
                    <target>1.6</target>
                to use 1.7. However it compiled successfully. Should we just leave it, or update it to 1.7 ?

Reply:
Hmm, too big of a chance for this... maybe just define the string as a constant in this class?

Reply:
yes i changed it to Charset.forName("UTF-8") it should be equals to it.